sample/package.json

```diff
@@ -6,8 +6,9 @@
   "license": "MIT",
   "scripts": {
     "test": "yarn hardhat test",
-    "deploy:metacoin": "yarn hardhat deploy --contract metacoin",
-    "deploy:greeter": "yarn hardhat deploy --contract greeter"
+    "reload": "cd .. && yarn build && cd sample && rm yarn.lock && rm -rf node_modules && yarn install",
```

Comment:
This is a utility function for quickly testing while iterating on changes in the root plugin project